conf/application.conf

```diff
@@ -17,6 +17,8 @@ include "frontend.conf"
 appName = "accessibility-statement-frontend"
 play.http.router = prod.Routes
 
+play.http.filters = "uk.gov.hmrc.accessibilitystatementfrontend.Filters"
```

Comment:
I think this might overwrite the bootstrap frontend filters that get loaded by default

I think we need to probably append the filter like is done for the CSPFilter below I think, so something like:

```
play.filters.enabled += "uk.gov.hmrc.accessibilitystatementfrontend.filters.LanguageChangeFilter"
```